docs/developer-guide/getting-started-with-persistence.md

```diff
@@ -75,173 +43,193 @@ following code snippet to your module's `pom.xml` under the `<dependencies>` sec
 </dependency>
 ```
 
-Notice that there is no need to define any direct dependencies on OrientDB or Spring Data - it's already done via 
+Notice that there is no need to define any direct dependencies on JanusGraph or Spring Data - it's already done via 
 the `strongbox-data-service` module.
 
 ## Creating Your Entity Class
 
 Let's now assume that you have a POJO and you need to save it to the database (and that you probably have at least 
-CRUD operation's implemented in it as well). Place your code under the `org.carlspring.strongbox.domain.yourstuff` 
-package. For the sake of the example, let's pick `MyEntity` as the name of your entity.
+CRUD operations implemented in it as well). Place your code under the `org.carlspring.strongbox.domain` 
+package. For the sake of the example, let's pick `PetEntity` as the name of your entity.
 
 If you want to store that entity properly you need to adopt the following rules:
 
-* Extend the `org.carlspring.strongbox.data.domain.GenericEntity` class to inherit all required fields and logic from 
-  the superclass.
-* Define getters and setters according to the `JavaBeans` coding convention for all non-transient properties in your 
-  class.
-* Define a default empty constructor for safety (even if the compiler will create one for you, if you don't define any 
-  other constructors) and follow the `JPA` and `java.io.Serializable` standards.
-* Override the `equals() `and `hashCode()` methods according to java `hashCode` contract (because your entity could be 
-  used in collection classes such as `java.util.Set` and if you don't define such methods properly other developers or 
-  yourself will be not able to use your entity).
-* _Optional_ - define a `toString()` implementation to let yourself and other developers see something meaningful in 
-  the debug messages.
+* Create the interface for your entity with all the getters and setters that are required to interact with the entity, according to the `JavaBeans` coding convention. This interface should extend `org.carlspring.strongbox.data.domain.DomainObject`. We need an interface in order to hide the implementation-specific details that depend on the underlying database, such as inheritance strategy.
+* Create the entity class which implements the above interface and extend to `org.carlspring.strongbox.data.domain.DomainEntity`.
+* Declare an entity class with `@NodeEntity` or `@RelationshipEntity`.
+* Define a default empty constructor, this would need to create entity instance from `neo4j-ogm` internals.
```

Comment:
Would you mind re-working this sentence? I'm not sure what you mean exactly. Is this what you're trying to say:
```
* Define a default empty constructor, as this would be required in order to create entity instances from `neo4j-ogm` internals.
```
?

Reply:
fixed

Reply:
Thanks! :)